app/mainAppWindow/index.js

```diff
@@ -53,20 +53,34 @@ exports.onAppReady = function onAppReady() {
 		window = null;
 	});
 
-	window.loadURL(config.url);
+	url = processArgs(process.argv)
+	window.loadURL( url ? url:config.url);
 
 	if (config.webDebug) {
 		window.openDevTools();
 	}
 }
 
-exports.onAppSecondInstance = function onAppSecondInstance(args) {
+exports.onAppSecondInstance = function onAppSecondInstance(event, args) {
 	console.log('second-instance started');
 	if (window) {
-		console.log('focusing on window');
-		if (window.isMinimized()) window.restore();
-		window.focus();
-		window.webContents.send('openUrl', args[0]);
+		event.preventDefault();
+		url = processArgs(args)
+		if (url) {
+			window.loadURL(url)
+		}
+	}
+}
+
+function processArgs(args) {
+	for (const arg of args) {
```

Comment:
is probably good to print the args object for debugging proposes.
```javascript
console.debug("processArgs", args);
```

I suspect there are a few other arguments that people can implement the handler for. Maybe guest access.

Reply:
Done!